site/css/style.css

```diff
@@ -1,50 +1,214 @@
+/* general styles */
 body {
   font-family: 'Source Sans Pro', sans-serif;
   font-weight: bold;
   margin: 0;
 }
+a li {
+  font-weight: bold;
+  color: black;
+}
 form {
   margin: 0;
   padding: 5px;
 }
+h3 {
+  color: 00B398;
+  display: inline-block;
+}
+.invite {
+  float: right;
+}
+input {
+  width: 100%;
+  font-size: 16px;
+}
+table {
+  width: 100%;
+}
+.buttonbar {
+  float: right;
+}
+.button {
+  width: 50%;
+  float:right;
+}
+.resource-details div {
+  text-align: left;
+  margin:10px 0;
+}
+
+/* page specific styles */
+.button input {
+  border: none;
+  outline: none;
+  background:lightsteelblue;
+  width: 210px;
+  border-radius: 3px;
+  margin: 5px;
+  padding: 5px;
+  float: right;
+}	
+textarea {
+  border: solid 1px lightgrey;
+  display: block;
+  width: 100%;
+}
+.buttonbar .usermgt {
+  width: 30%;
+  float: right;
+  margin: 8px;
+}
+.buttonbar .usermgt input {
+  width: 100%;
+  margin: 5px;
+}
+.small-input {
+  width: unset;
+}
+.onefield {
+  margin: 15px 0;
+  text-align: right;
+}
+.password-label {
+  display: inline-block;
+  width: 40%;
+}
+.singleton {
+  vertical-align: top;
+}
+.password {
+  width: 25%;
+  outline: 1px solid lightgrey;
+  vertical-align: super;
+}
+th {
+  text-align: left;
+}
+.user-details th, .user-details td {
+  padding: 5px;
+  text-align: right;
+}
 a {
```

Comment:
it looks like there was a redundant style assignment (color) but it makes sense to distinguish two different rules